features/media-queries-update.yml

```
@@ -0,0 +1,6 @@
+name: Update frequency media feature
+description: The `update` CSS media feature sets styles based on whether and how fast the user's device can modify display after it has been rendered. Values are `none`, `slow`, and `fast`.
```

```suggestion
description: The `update` CSS media feature sets styles based on whether and how fast the user's device can modify display after it has been rendered.
```

I feel like enumerating the values isn't needed here. The first sentence already encapsulates the information that this can be used to detect devices that don't refresh at all, and devices that refresh slow or fast.